src/MIDebugEngine/Engine.Impl/ExceptionManager.cs

```diff
@@ -289,15 +289,16 @@ public bool TryGetExceptionBreakpoint(string bkptno, out string exceptionName, o
                 ulong breakpointNumber = Convert.ToUInt32(bkptno, CultureInfo.InvariantCulture);
                 lock (categorySettings.CurrentRules)
                 {
-                    if (categorySettings.CurrentRules.ContainsValue(breakpointNumber))
+                    exceptionName = categorySettings.CurrentRules.FirstOrDefault(pair => pair.Value == breakpointNumber).Key;
```

Comment:
You need `?.Key`

Reply:
I tried this before removing the `?`, but received the below error:
![image](https://user-images.githubusercontent.com/60245970/143963209-bee1bd39-5697-4619-baae-3894e898a033.png)

Reply:
Oh, I am sorry.